ext/datasource/k8s/deploy/crds/datasource.sentinel.com_circuitbreakerrules_crd.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cbrules.datasource.sentinel.com
```

Comment:
The abbr might not be clear enough. Maybe cbrules -> `circuitbreaker.rules` or `circuitbreakerrules` or other better representation?

Reply:
In open-source world we'd like `*.sentinel.io` (though the domain is not owned by us :(

Reply:
I prefer circuitbreaker.rules